framework/src/Volo.Abp.AspNetCore.Components.Web.Theming/Layout/StandardLayouts.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Volo.Abp.AspNetCore.Components.Web.Theming.Layout;
+
+public static class StandardLayouts
+{
+    public const string Main = "Main";
+    public const string TopMenu = "TopMenu";
+    public const string SideMenu = "SideMenu";
```

Comment:
We shouldn't use it here. It's a theme-specific option. We can move `TopMenu` and `SideMenu` option to the LeptonX project.

Also `Main` should be renamed as `Application` and this file can contain only 1 layout since blazor has only Application layout